.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5502
```

Comment:
If we don't want to save this file to the repo, you can add it to `.gitignore`.

Reply:
Like Steve said, we should exclude this file as it's a code editor configuration